components/TipDetails/index.js

```diff
@@ -16,11 +19,10 @@ export default function TipDetails({ tip }) {
         <StyledDescription>{tip.bodyContent}</StyledDescription>
       </StyledTipContainer>
 
-      <Link href="/plantTipsPage">
-        <StyledIconContainer>
+     
+        <StyledIconContainer onClick={() =>  router.back()}>
```

Comment:
Replace the `span` with a `button` or `Link` for better semantic HTML and accessibility.

A` button` is intended for actions (like going back), and a `Link` is intended for navigation. These elements are inherently accessible and come with built-in keyboard support.